django_app/redbox_app/templates/base.html

```diff
@@ -76,6 +76,24 @@
     <div class="govuk-width-container">
       <div class="govuk-footer__meta">
         <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+          <h2 class="govuk-visually-hidden">Support links</h2>
```

Comment:
Side note in this doc, can we update `Up to UK OFFICIAL SENSITIVE documents can be used in this tool` to say explicitly whether it includes or doesn't include `OS`

Reply:
I'll make a note to check this separately